src/prefect/environments/execution/dask/k8s.py

```diff
@@ -322,8 +326,11 @@ def _extra_loggers(self) -> str:
         cluster_loggers = [
             "dask_kubernetes.core",
             "distributed.deploy.adaptive",
-            "kubernetes",
         ]
+
+        if self.log_k8s_erros:
```

Comment:
I'm super impressed that you managed to perfectly replicate this typo in exactly two places so that it's still functional and passes tests
```suggestion
        if self.log_k8s_errors:
```

Reply:
🤦